<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LensControl_Optotune</class>
 <widget class="QWidget" name="LensControl_Optotune">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LensControl_Optotune</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>450</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Optotune Lens Control</string>
   </property>
   <widget class="QWidget" name="centralWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>401</width>
      <height>221</height>
     </rect>
    </property>
    <widget class="QPushButton" name="File_load">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="File_display">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>271</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>None</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="Status_text">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>191</width>
       <height>131</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Status_back">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Status_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>66</y>
       <width>120</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Testing</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Trigger</string>
     </property>
     <widget class="QCheckBox" name="Trigger_adwin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>ADwin</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="Continous">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>135</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Continous</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Singal">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Singal</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="Progress">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>271</width>
       <height>13</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="Stop">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="Temperature">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>T: NA</string>
     </property>
    </widget>
    <widget class="QLabel" name="Focus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>200 mm</string>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="Focus_Slider">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="maximum">
     <number>250</number>
    </property>
    <property name="value">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>280</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>250 mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>50 mm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Connect">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Reset">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
